### PR TITLE
Use understandable names for breakpoint control and address registers

### PR DESCRIPTION
**Pull request by fjricci**

### CODE REVIEW

File: Sources/Host/Linux/ARM/PTraceARM.cpp [hunk 1]
```diff
@@ -115,26 +115,26 @@ ErrorCode PTrace::readCPUState(ProcessThreadId const &ptid, ProcessInfo const &,
...
 
     if (wrapPtrace(PTRACE_GETHBPREGS, pid, (n << 1) + 1, &value) < 0) {
-      state.hbp.bcr[n] = 0;
+      state.hbp.bp_addr[n] = 0;
     } else {
-      state.hbp.bcr[n] = value;
+      state.hbp.bp_addr[n] = value;
     }
 
     if (wrapPtrace(PTRACE_GETHBPREGS, pid, (n << 1) + 2, &value) < 0) {
```
Comment (sas) [inline Sources/Host/Linux/ARM/PTraceARM.cpp]: I wonder why we use a temp variable here (`value`) and we don't for watchpoints?

**Comment by sas** [comment]
@facebook-github-bot shipit

File: Sources/Host/Linux/ARM/PTraceARM.cpp (hunk 1 above)
Comment (fjricci) [inline Sources/Host/Linux/ARM/PTraceARM.cpp]: I'm currently overhauling this pretty significantly currently. So I will make it more consistent as I go :smiley:

**Comment by fjricci** [comment]
Clang-formatted and committed as fe86030c2597cc4a9769ca6de0fbedc0618330e7

**Pull request closed without merging**